netbox/dcim/views.py

```diff
@@ -122,14 +122,18 @@ def post(self, request):
             if form.is_valid():
 
                 with transaction.atomic():
-
+                    cables = []
```

Comment:
We can use a set (rather than a list) for this to automatically ignore duplicated IDs.
```suggestion
                    cables = set()
```